doc/API.md

```diff
@@ -110,14 +111,16 @@ Creates a new API client.
 ### Request
 ___Parameters___
 * name - API client name.
+* redirectUrl (optional) - API client redirection URL for user authentication. Clients that require user authentication should provide a redirection URL.
 
 ```ssh
 POST /api/clients HTTP/1.1
 Content-Type: application/json
 Cache-Control: no-cache
 Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6ImFkbWluIiwic2NvcGUiOiJhZG1pbiIsImlhdCI6MTQ3NDY0Nzc1OX0.R1vQOLVg8A-6i5QaZQVOGAzImiPvgAdkWiODYhYiNn4
 {
-	"name": "SensorWebClient"
+	"name": "SensorWebClient",
+  "redirectUrl": "https://domain.org/auth/complete"
```

Comment:
nit: looks like an indentation mismatch; I don't have any preference with/without, up to you :)